src/System.Diagnostics.EventLog/tests/EventLogReaderTests/EventLogReaderTests.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Globalization;
+using System.IO;
+using Xunit;
+using System.Diagnostics.Eventing.Reader;
+
+namespace System.Diagnostics.Tests
+{
+    public class EventLogReaderTests
+    {
+        [ConditionalFact(typeof(Helpers), nameof(Helpers.SupportsEventLogs))]
+        public void CreateReader()
+        {
+            // var eventLog = new EventLogReader("XXX");
```

Comment:
Looks like tests are yet to be completed.

Reply:
Yes we have a deal with the community that we need them to step up and do the tests before we check in see https://github.com/vancem/corefx/pull/1